src/ethereum/EthereumNetwork.ts

```diff
@@ -346,71 +396,15 @@ export class EthereumNetwork {
         // batch token balance queries.
         !isFetchTokenBalancesSupported
       ) {
-        await this.checkAndUpdate(
-          this.ethNeeds.tokenBalLastChecked[currencyCode] ?? 0,
-          BAL_POLL_MILLISECONDS,
-          async () => await this.check('fetchTokenBalance', currencyCode)
-        )
+        this.acquireTokenBalance(currencyCode)(currencyCode).catch(error => {
+          console.error(error)
```

Comment:
The old `checkAndUpdate` function would call `this.ethEngine.error('checkAndUpdate ', ...`. Shouldn't we continue to report errors in the same way?